pages/sponsor/prospectus.vue

```diff
@@ -38,42 +64,172 @@
                     $t(`terms.${level}`)
                 }}</router-link>
             </li>
-        </ul>
+        </ul> -->
 
         <!-- 2020 stats -->
-        <h2>{{ $t('stats.0') }}</h2>
-        <ul class="list-disc">
-            <li
-                v-for="(item, i) in $t('stats.1')"
-                :key="`prospectus_stats_${i}`"
-            >
-                {{ item }}
-            </li>
-        </ul>
-
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('statsTitle') }}</p>
+            </template>
+            <template #right-col>
+                <ul class="list-disc ml-4">
+                    <li
+                        v-for="(item, i) in $t('stats')"
+                        :key="`prospectus_stats_${i}`"
+                    >
+                        {{ item }}
+                    </li>
+                </ul>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
         <!-- important dates -->
-        <h2>{{ $t('importantDates.0') }}</h2>
-        <i18n path="importantDates.1" tag="p"></i18n>
-        <ul class="list-disc">
-            <i18n
-                v-for="(date, i) in $t('importantDates.2')"
-                :key="`prospectus_dates_${i}`"
-                :path="`importantDates.2.${i}`"
-                tag="li"
+        <two-col-wrapper>
+            <template #default>
+                <p>{{ $t('importantDatesTitle') }}</p>
+                <!-- <i18n path="importantDatesSubtitle" tag="p"></i18n> -->
+            </template>
+            <template #right-col>
+                <ul class="list-disc ml-4">
+                    <i18n
+                        v-for="(date, i) in $t('importantDates')"
+                        :key="`prospectus_dates_${i}`"
+                        :path="`importantDates.${i}`"
+                        tag="li"
+                    >
+                        <template #AoE>
+                            <ext-link
+                                href="https://www.timeanddate.com/worldclock/converter.html?iso=20210427T115900&p1=tz_aoe&p2=241&p3=1440"
+                                highlight
+                                >AoE</ext-link
+                            >
+                        </template>
+                    </i18n>
+                </ul>
+            </template>
+        </two-col-wrapper>
+        <core-hr></core-hr>
+        <!-- sponsor package mobile -->
+        <p class="section-title text-sm md:text-lg lg:hidden">
+            {{ $t('package-title.0') }}
+        </p>
+        <div class="grid mt-8 lg:hidden">
+            <div
+                v-for="(sponsorPackages, i) in sponsorLevels"
+                :key="`sponsor_packages_${i}`"
+                class="grid sponsor-packages gap-x-3 mt-2"
+            >
+                <div class="section-title">
+                    <div class="mt-24 md:mt-36">
```

Comment:
consider adding a dummy `div` for the margin and using v-for looping through the titles (also for the special sponsorship part). e.g.:
```
<div class="mt-24 md:mt-36"></div>
<p v-for="i in XXX" class="text-xs md:text-sm">
    {{ $t(`package-title.${i}`) }}
</p>
```